tools/docsbuilder/README.md

```diff
@@ -0,0 +1,40 @@
+# Docs Builder
+
+## Overview
+
+This project is used for building and pushing documentation-related Docker images. They are built from sources (`/docs` directory on this repository).
+
+> **Note:** This is a temporary solution, before migration to the new Documentation Delivery concept. 
+
+## Prerequisites
+
+Use the following tools to set up the project:
+
+* [Go distribution](https://golang.org)
+* [Docker](https://www.docker.com/)
+
+## Development
+
+### Install dependencies
+
+This project uses `dep` as a dependency manager. To install all required dependencies, use the following command:
+```bash
+dep ensure -vendor-only
+```
+
+### Run
+
```

Comment:
We need a sentence here, for example:
`To run this tool, run the following command:`